corehq/apps/export/models/new.py

```diff
@@ -1390,7 +1390,7 @@ def generate_schema_from_builds(cls, domain, app_id, identifier, force_rebuild=F
         try:
             current_schema = cls._reorder_schema_from_app(current_schema, app_id, identifier)
         except Exception as e:
-            _soft_assert = soft_assert('{}@{}'.format('brudolph', 'dimagi.com'))
+            _soft_assert = soft_assert(notify_admins=True)
```

Comment:
usually i treat it as a 500 to fix, but don't want it to crash for a user. but agree probably not that useful for people